en/docs/architecture-diagram/architecture-diagram.md

```diff
@@ -69,62 +62,77 @@ isolated function getSupportedCurrencies() returns string[]|error {
+![Extension Command](../img/design-diagrams.png)
 
-This diagram displays all the services available within the workspace with directed links representing any interactions between them. The Service Interaction Diagram generated for the 10 Ballerina packages in the Google Cloud Platform demo project is shown below.
+### Service Diagram: Level 1
+
+This diagram displays all the services available within the workspace with links representing any interactions between them. The basic Service Diagram (level 1) generated for the 10 Ballerina packages in the Google Cloud Platform demo project is shown below.
 
 ![Service Interaction Diagram](../img/pdd-service-interaction-diagram.png)
->**Info:** The directed link between the `Checkout` service and the `Email` service is an indication that a component of the `Checkout` service invokes the `Email` service. In addition to this, the `Email` service also invokes an `external` service via a connector.
 
-#### Detailed diagram
-The detailed Service Interaction Diagram (Level 2) dives deeper into the compositions of the services and the interactions between them. The following diagram is the detailed representation of the demo project, and it depicts the individual resource/remote functions of the services and their invocations.
+>**Info:** The link between the `Checkout` service and the `Email` service indicates that a component of the `Checkout` service invokes the `Email` service. In addition to this, the `Email` service also invokes an `external` service via a connector.
+
+### Service Diagram: Level 2
+
+The detailed Service Diagram (Level 2) dives deeper into the compositions of the services and the interactions between them. Shown below is the detailed representation of the demo project, which depicts the individual resource/remote functions of the services and their invocations.
 
 ![Detailed Service Interaction Diagram](../img/pdd-detailed-service-interaction-diagram.png)
 
 The data types of the request and response bodies of the interactions can be viewed by hovering over a particular invocation.
 
 ![Detailed Service: Request Response Data Popups](../img/pdd-request-response-data-popup.png)
 
-### Types Diagram
+### Type Diagram
 
 The types diagram provides a comprehensive view of all the Ballerina record types defined in the project workspace and their associations.
 
-   ![Types Diagram](../img/pdd-types-diagram.png)
+![Types Diagram](../img/pdd-types-diagram.png)
 
-   >**Info:** The multiplicities of the associations are represented on either side of the connector while the [type inclusions](https://ballerina.io/learn/by-example/type-inclusion-for-records/) are differentiated using a directed link (represents inheritance).
+>**Info:** The multiplicities of the associations are represented on either side of the connector while the [type inclusions](https://ballerina.io/learn/by-example/type-inclusion-for-records/) are differentiated using a directed link (represents inheritance).
```

Comment:
```suggestion
>**Info:** The multiplicities of the associations are represented on either side of the connector, while the [type inclusions](https://ballerina.io/learn/by-example/type-inclusion-for-records/) are differentiated using a directed link (represents inheritance).
```

Reply:
Do we really need the comma here? Will add it anyway if it feels more clear. I think it works with or without.

Another thing I missed - AFAIU, `multiplicities of associations` is a coding specific that users will understand. Hope that is correct. 

Otherwise, the word `multiplicities` is very weird. :)

Reply:
Usually it is referred to as `Multiplicity`. https://docs.oracle.com/cd/E19159-01/819-3669/bnbqh/index.html